tools/clang/test/lit.cfg

```diff
@@ -503,6 +503,11 @@ if config.enable_backtrace == "1":
 if config.spirv:
     config.available_features.add("spirv")
 
+build_mode = lit_config.params.get('build_mode', None)
+
+if build_mode == "Release" or build_mode == "RelWithDebInfo":
+    config.available_features.add("release_build")
```

Comment:
This change isn't needed if using `UNSUPPORTED: asserts` instead, which I think is more correct.

Reply:
Updated.